deployment/clouddeploy/gke-workers/environments/oss-vdb-test/importer-deleter.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: importer-deleter
+spec:
+  schedule: "* */6 * * *"
+  concurrencyPolicy: Forbid
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          tolerations:
+          - key: workloadType
+            operator: Equal
+            value: default-pool
+          containers:
+          - name: importer-deleter
+            env:
+            - name: GOOGLE_CLOUD_PROJECT
+              value: oss-vdb-test
+            image: importer
+            args:
+              - --delete
+              - --delete_threshold_pct=10
+              - --public_log_bucket=osv-test-public-import-logs
+            imagePullPolicy: Always
+            volumeMounts:
+              - mountPath: "/work"
+                name: "ssd"
+            securityContext:
+              privileged: true
+            resources:
+              requests:
+                cpu: 1
+                memory: "10G"
+              limits:
+                cpu: 1
+                memory: "11G"
+          nodeSelector:
+-           cloud.google.com/gke-nodepool: default-pool
```

Comment:
This also isn't explicitly needed - all the other node pools have taints so it will be de facto scheduled on the default-pool

Reply:
Removed in 71dcb05b